pkg/browser/browser.go

```diff
@@ -30,7 +30,11 @@ func ForOS(goos, url string) *exec.Cmd {
 		r := strings.NewReplacer("&", "^&")
 		args = append(args, "/c", "start", r.Replace(url))
 	default:
-		exe = "xdg-open"
+		if findExe("wslview") {
```

Comment:
Sorry to be super nitpicky about this 😅 But now the logic prefers `wslview` if it exists, i.e. it is no longer a fallback. I'm thinking that `xdg-open` should have precedence on Linux, WSL or not.

To reiterate my ask from earlier: `wslview` (if it exists) as fallback if `xdg-open` doesn't exist.